Doc/c-api/init.rst

```diff
@@ -19,6 +19,12 @@ a few functions and the :ref:`global configuration variables
 
 The following functions can be safely called before Python is initialized:
 
+* Functions that initialize the interpreter:
+
+  * :c:func:`Py_Initialize`
+  * :c:func:`Py_InitializeEx`
+  * :c:func:`Py_Main`
```

Comment:
I may be nice to also document Py_BytesMain().

Reply:
Yeah, this was a super-old PR that I had actually forgotten about - it predates PEP 587 and the other improvements that landed for 3.8.

I'll look at adding the new APIs here when I fix the conflicts (as I assume the conflicts are with the docs for the new APIs).

That won't be tonight though - maybe Sunday.

Reply:
Yep, the conflict in veryhigh.rst was indeed with Py_BytesMain - I've moved that over to the init section, and added it to the pre-init safe list.